src/styles.css

```diff
@@ -1,3 +1,46 @@
 .invisible {
     display: none;
+}
+
+.navbar {
+    background-color: rgb(255, 38, 0);
+    height: 12vh;
+    border-bottom-left-radius: 10%;
+    border-bottom-right-radius: 10%;
+}
+
+.nav-link {
+    font-size: 20px;
+    color: black;
+    transition: 0.2s;
+}
+
+.nav-link:visited,
+.current,
+.nav-link:hover {
+    color: black;
+    background-color: rgb(196, 29, 0);
+    border-radius: 15px;
+}
+
+.block {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 20vw;
+    border-radius: 10px;
+    height: max-content;
+    background-color: rgb(255, 38, 0) ;
```

Comment:
- [ ] [OPTIONAL] The red colour looks a little weird. I suggest you change it to something dimmer for a better user experience.